ts/utils/paypal.ts

```diff
@@ -0,0 +1,8 @@
+import { PayPalInfo } from "../../definitions/pagopa/PayPalInfo";
+import I18n from "../i18n";
+
+// return the email used in the paypal account. If it can't be found "n/a" will be returned
+// it is inside the default psp ¯\_(ツ)_/¯ that could be one in the relative array 'pspInfo'
+export const getPaypalAccountEmail = (paypal: PayPalInfo): string =>
```

Comment:
The PR lgtm what about adding some quick test on this function?

Reply:
Sure
I'm going to add some tests

Reply:
Added https://github.com/pagopa/io-app/pull/3527/commits/6cbd5fc8bbb015f9d6e3e2086f24e1fb00e559e5